share/smack/svcomp/utils.py

```diff
@@ -42,6 +44,15 @@ def svcomp_frontend(args):
   smack.top.clang_frontend(args)
 
 def svcomp_process_file(args, name, ext):
+  # Check if property is vanilla reachability, and return unknown otherwise
+  if args.svcomp_property:
+    with open(args.svcomp_property, "r") as f:
+      prop = f.read()
+    if "valid-deref" in prop:
```

Comment:
@zvonimir I am not sure, I think this is from your commit https://github.com/smackers/smack/commit/4c2a32bbad1bb02357da2cf21131821d984eb732